README.md

```diff
@@ -422,12 +422,47 @@ with at least one node that has persistent memory device(s). For development or
 testing, it is also possible to use a cluster that runs on QEMU virtual
 machines, see the ["QEMU and Kubernetes"](#qemu-and-kubernetes) section below.
 
+- **Make sure that alpha feature gates CSINodeInfo, CSIDriverRegistry are enabled**
+
+The method to configure alpha feature gates may vary, depending on the kubernetes deployment.
+
 - **Label the cluster nodes that provide persistent memory device(s)**
 
 ```sh
     $ kubectl label node <your node> storage=pmem
 ```
 
+- **Verify that the node labels have been configured correctly**
+
+```sh
+    $ kubectl get nodes --show-labels
+```
+
+Output of above command should indicate that every non-master node has labels:
```

Comment:
Not just some labels, but at least these two. I suggest to say:
"that every node with PMEM has these two labels:"

Reply:
How do we treat master here? My intention with spelling out "non-master" was to leave out master in that point. If yoy spell it "every node with PMEM", do you also include master in that set, even if you mean _node_, like implicitly excluding master? Because ""kubectl get nodes" shows nodes including master, and master is special, it does not have node= label, but has master= label. Also, in our test cluster with VMs, we will violate our own policy then because master has PMEM device but does not have storage=pmem label. I noticed that in govm work, current clear-based script specifically tries to remove storage= label from master, but in VM config we have the device. Cleaner would be to add storage=pmem label on master as well, expressing phys. config. K8s will not schedule driver there anyway, right, or does it.

Reply:
The "master" node is only special if the cluster admin decides that it is, for example by leaving the taint so that normal  pods don't run there. For this part of the documentation we don't care about whether master is special, all that matters is whether the node has a PMEM region.

> Also, in our test cluster with VMs, we will violate our own policy then because master has PMEM device but does not have storage=pmem label.

No, the master node has no PMEM device:
```
qemu-system-x86_64 -enable-kvm -bios /nvme/gopath/src/github.com/intel/pmem-csi/_work/OVMF.fd -smp sockets=1,cpus=4,cores=2 -cpu host -m 2048 -vga none -nographic -object rng-random,filename=/dev/urandom,id=rng0 -device virtio-rng-pci,rng=rng0 -drive file=_work/clear-kvm.0.img,if=none,aio=threads,id=disk -device virtio-blk-pci,drive=disk,bootindex=0 -drive file=/nvme/gopath/src/github.com/intel/pmem-csi/_work/clear-kvm.0.img.data,if=none,aio=threads,id=data,format=raw -device virtio-blk-pci,drive=data -netdev tap,ifname=pmemcsitap0,script=no,downscript=no,id=mynet0 -device virtio-net-pci,netdev=mynet0,mac=DE:AD:BE:EF:01:00 -debugcon file:_work/clear-kvm.0.img.debug.log -global isa-debugcon.iobase=0x402 -m 2048M -monitor none -serial file:_work/clear-kvm.0.log
```

Only the worker nodes do:
```
qemu-system-x86_64 -enable-kvm -bios /nvme/gopath/src/github.com/intel/pmem-csi/_work/OVMF.fd -smp sockets=1,cpus=4,cores=2 -cpu host -m 2048 -vga none -nographic -object rng-random,filename=/dev/urandom,id=rng0 -device virtio-rng-pci,rng=rng0 -drive file=_work/clear-kvm.2.img,if=none,aio=threads,id=disk -device virtio-blk-pci,drive=disk,bootindex=0 -drive file=/nvme/gopath/src/github.com/intel/pmem-csi/_work/clear-kvm.2.img.data,if=none,aio=threads,id=data,format=raw -device virtio-blk-pci,drive=data -netdev tap,ifname=pmemcsitap2,script=no,downscript=no,id=mynet0 -device virtio-net-pci,netdev=mynet0,mac=DE:AD:BE:EF:01:02 -debugcon file:_work/clear-kvm.2.img.debug.log -global isa-debugcon.iobase=0x402 -m 2048M,slots=2,maxmem=34816M -object memory-backend-file,id=mem1,share=on,mem-path=_work/clear-kvm.2.pmem.raw,size=32768M -device nvdimm,id=nvdimm1,memdev=mem1,label-size=2097152 -machine pc,nvdimm -monitor none -serial file:_work/clear-kvm.2.log
```

Reply:
yes I was wrong, I mislooked "if not host-0" part in qemu flags setup